src/drivers/ist8310/ist8310.cpp

```diff
@@ -1249,164 +1215,6 @@ IST8310::set_selftest(unsigned enable)
 	return !(str == str_reg_ret);
 }
 
-bool
```

Comment:
It's not called by anybody. If there's a way to enable it, it should be done for every user of ist8310 and at the very least a runtime detection to check if the sensor supports it.